python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,135 @@
+plugin:
+  name: Generate PlantUML Diagrams
+  package: aac.plugins.puml
+  description: |
+    An AaC plugin that enables PlantUML diagrams to be generated from AaC
+    definitions.
+  commands:
+    - name: puml-component
+      help_text: Converts an AaC-defined system to a PlantUML component diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML component diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Component Diagram Test
+        scenarios:
+          - name: Output PlantUML component diagram for valid architecture.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid file path.
+            when:
+              - The aac app is run with the puml-component command.
+            then:
+              - A PlantUML component diagram of the model is written to the output directory.
+          - name: Output PlantUML component diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid file path.
+            when:
+              - The aac app is run with the puml-component command.
+            then:
+              - A PlantUML component diagram of the model is generated to a .puml file.
+    - name: puml-sequence
+      help_text: Converts an AaC-defined usecase to PlantUML sequence diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined usecase from which
+            to generate a PlantUML sequence diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Sequence Diagram Test
+        scenarios:
+          - name: Output PlantUML sequence diagram for valid use case.
```

Comment:
Looks like the requested update to the scenario name did not get brought in, think it is helpful to have an explicit name like the second scenario cases. 

```suggestion
          - name: Output PlantUML sequence diagram for valid use case to requested output directory.
```